releases/latest/kernel/helpers/build/populate_scc.sh

```diff
@@ -89,7 +92,10 @@ do
   /opt/ol/wlp/bin/server start && /opt/ol/wlp/bin/server stop
 done
 
-rm -rf /output/messaging /logs/* $WLP_OUTPUT_DIR/.classCache && chmod -R g+rwx /opt/ol/wlp/output/*
```

Comment:
are you sure we don't need `chmod -R g+rwx /opt/ol/wlp/output/*` anymore?   CC @arturdzm

Reply:
Assuming that all files created by the server will have `rw` permissions not just `r` I think its ok. There could be files copied by the user into that folder though that are not readable/writable by the group. Eg would be keystore p12 file. That depends on what permissions the files had on a host